src/main/java/edu/iis/mto/time/Order.java

```diff
@@ -5,14 +5,21 @@
 import java.util.ArrayList;
 import java.util.List;
 
+
 public class Order {
 
     private static final long VALID_PERIOD_HOURS = 24;
     private State orderState;
     private List<OrderItem> items = new ArrayList<>();
     private LocalDateTime subbmitionDate;
+    private LocalDateTime orderDate;
 
     public Order() {
+        orderDate =LocalDateTime.now();
+        orderState = State.CREATED;
+    }
+    public Order(LocalDateTime localDateTime) {
```

Comment:
chodziło o to, aby móc manipulować zegarem (Clock) a nie wprowadzać określoną datę. Tu przypadkowo to zadziała, ale gdyby test dotyczył daty "do przodu" (np z ograniczeniem miesiąca) to po jakimś test przestałby działać.